ui/admin/tests/acceptance/host-catalogs/hosts/update-test.js

```diff
@@ -108,68 +125,82 @@ module('Acceptance | host-catalogs | hosts | update', function (hooks) {
           details: {
             request_fields: [
               {
-                name: 'name',
-                description: 'Name is required.',
+                name: 'address',
```

Comment:
Changed the field we mark as invalid. `name` is an optional field within hosts, but address is required one, so this change is to make the test realistic.